aries-backchannels/javascript/server/src/TestHarnessConfig.ts

```diff
@@ -55,6 +59,25 @@ export class TestHarnessConfig {
     const agentArgs = await this.getAgentArgs(options)
 
     this._agent = await createAgent(agentArgs)
+
+    try {
+      const key = await this._agent.context.wallet.createKey({keyType: KeyType.Ed25519, privateKey: TypedArrayEncoder.fromString(agentArgs.publicDidSeed)})
```

Comment:
I was thinking we can now do `agent.wallet.createKey`, but that's in the open PR... nvm